scss/footer.scss

```diff
@@ -10,24 +11,22 @@
   flex-direction: column;
   height: 180px;
   padding: 40px 20px;
+}
 
-  .game-changers-text {
-    @include white-text;
-    $text-size: 20px;
-    font-size: $text-size;
-    line-height: $text-size;
-  }
+.game-changers-text {
+  @extend %white-text;
+  font-size: $game-changers-font-size;
+  line-height: $game-changers-font-size + 3;
```

Comment:
I dont know if it is the correct way use the plain numbers intead a constant. We will wait until @mcallegari10 check it :+1: